source/_static/scss/includes/_layout.scss

```diff
@@ -31,4 +31,5 @@
     flex-shrink: 0;
     position: sticky;
     top: 0;
+    max-width: 200px;
```

Comment:
Given a 1366 resolution, I'd like to keep at least 600 for the main body, so restricting this should guarantee that (?).